### DISCUSSION TITLE
scroll restoration

### DISCUSSION THREAD

**Discussion by xqfd007** [vercel/next.js#58906] [Ideas]
### Goals

Hello, I would like to ask a question about Next.js 14. In a PLP (Product List Page), when navigating to a PDP (Product Detail Page) and then going back to the PLP, how is the scroll restoration achieved? 

### Non-Goals

1.
2.
3.

### Background

Hello, I would like to ask a question about Next.js 14. In a PLP (Product List Page), when navigating to a PDP (Product Detail Page) and then going back to the PLP, how is the scroll restoration achieved?

### Proposal

How to achieve

**Comment by duncanbla974** [comment]
Same here, any update ?
